flocker/control/_model.py

```diff
@@ -6,7 +6,7 @@
 """
 
 from characteristic import attributes, Attribute
-from pyrsistent import pmap
+from pyrsistent import pmap, pset
```

Comment:
That's probably useful eventually.  I'm happy to see pset used but I guess I'll be sad if we randomly use frozenset in some places and pset in others.  Maybe just stick with frozenset until pset is actually a win for something?